docs/user/editions/budgie/configuration.md

```diff
@@ -241,50 +239,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
 
 ## Desktop Icons
 
-With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Super + A`. Next, click the cog icon for Budgie Settings.
```

Comment:
These "Desktop Icon" section changes conflict with the changes we made yesterday. I would suggest you complete the any additional changes, like my review items, then try your hand at solving the conflict.

Reply:
I think I've got the specific consistency fixes in place now, and I think I was able to resolve the conflicts I created by basing these changes off of the original "master" branch rather than basing them off of the branch I had used to submit the prior PR (or simply adding to that branch to update the PR with more content)?